### DISCUSSION TITLE
I cannot use Control + Click or Shift + Click in this window.

### DISCUSSION THREAD

**Discussion by laramaceo** [tnodir/fort#313] [General]
There are areas in Fort where you can use Control + Click to select multiple items in a list, but not here. You also cannot use Shift + Click to select a range of contiguous items. 

Can these options be implemented in this window?

![Idea](https://github.com/user-attachments/assets/b7fb93c0-558a-47c1-a704-012889664469)

**Comment by tnodir** [comment]
Yes, it can be implemented. But why?

Usually only a couple of services should be made trackable..

**Reply by laramaceo** [reply]
Thank you for your reply, I am learning with it. I must find documentation and learn about how a firewall works. For now, I am using Fort in a very basic way.

**Reply by laramaceo** [reply]
I had not yet taken the time to review the Wiki section.
